src/pages/components/search/style.mdx

```diff
@@ -62,12 +62,22 @@ content.
 
 | Class                                             | Property                    | px / rem | Spacing token |
 | ------------------------------------------------- | --------------------------- | -------- | ------------- |
-| `.bx--search--lg`                                 | height                      | 48 / 3   | –             |
-| `.bx--search--sm`                                 | height                      | 32 / 2   | –             |
 | `.bx--search-magnifier` <br/> `.bx--search-close` | height, width               | 16 / 1   | –             |
 | `.bx--search--lg`                                 | padding-left, padding-right | 48 / 3   | `$spacing-09` |
 | `.bx--search--sm`                                 | padding-left, padding-right | 32 / 2   | `$spacing-07` |
 
+## Size
+
+The width of the search field should appropriately fit the design and layout of
+content.
+
+| Class                                             | Height(px / rem)                    | 
+| ------------------------------------------------- | ----------------------------------- | 
+| Small (sm)                                        | 32 / 2                              |    
+| Large (lg)                                        | 48 / 3                              |
```

Comment:
- There needs to be a Medium (md) 40 / 2.5 size. There is no Extra Large (xl) size.
- There needs to be a space between Height and (px / rem).